README.md

```diff
@@ -1,6 +1,32 @@
-# Ember-let
-
-This README outlines the details of collaborating on this Ember addon.
+# ember-let (WIP)
+
+`ember-let` is an addon for binding variables to template contexts in Ember. It behaves much like the `with` helper, but lets you bind an arbitrary number of variables, including standalone values, hashes, and class instances.
+
```

Comment:
I'd also mention that unlike `with` it will yield its block even if the bound values are `undefined` `null` or `[]`